crates/tests-e2e/examples/fund.rs

```diff
@@ -36,18 +36,31 @@ pub struct Opts {
     /// Maker address
     #[clap(long, default_value = "http://localhost:18000")]
     pub maker: String,
+
+    #[clap(long, default_value = "false")]
+    check_peer_alias: bool,
```

Comment:
Are you sure the channel is actually usable (and the funding worked) if the channel is active? 
We found that we can only reliably assume the channel is working if the peer id is set.

Reply:
this is often the case. I thought that at least making sure the channel is active is a step in the right direction.

This boolean flag now just ensures that we don't fail when peer alias is not set (e.g. when you want to locally pay the invoice from the maker, not from the faucet). It still unblocks me (and anyone using `just all`) , and gives the opt-in reliability for anyone who would like to have the strict behaviour via command line flag.